proxy.adoc

```diff
@@ -453,7 +453,7 @@ This configures the following:
 |zimbraMailReferMode |reverse-proxied.
 |zimbraMailPort |8080 (to avoid port conflicts)
 |zimbraMailSSLPort |8443 (to avoid port conflicts)
-|zimbraReverseProxyLookupTarget to |TRUE
+|zimbraReverseProxyLookupTarget |TRUE
 |TRUE
```

Comment:
From what I can tell, that's still only part of the cleanup... this line should be deleted altogether to keep from having table elements to be shifted by one causing keys and values to be reversed from here down in this table.